src/app/pages/home/home.page.ts

```diff
@@ -7,9 +11,62 @@ import { Component, OnInit } from '@angular/core';
 })
 export class HomePage implements OnInit {
 
-  constructor() { }
+  constructor(
+    private localStorage: LocalStorageService,
+    private http: HttpService,
+    private router: Router,
+    ) { }
 
+  projects: any;
+  name: any;
+  started: any = 0;
+  notStarted: any = 0;
+  chartData: any;
+  
+  async ionViewWillEnter(){
+    this.makeApiCall();
+  }
+  async makeApiCall() {
+    let data =  await this.localStorage.getLocalData(localKeys.USER_DETAILS)
+    const authToken = JSON.parse(data).access_token;
+    this.name = JSON.parse(data).user.name;
+    const headers = new HttpHeaders({
```

Comment:
To get header part write one common method in api service file so we can easily resuable that.